app/src/main/java/ohmyquiz/models/RegisterModel.java

```diff
@@ -1,30 +1,28 @@
-package ohmyquiz.models;
-
-public class RegisterModel {
-    private String username;
-    private String password;
-
-
-    public String getUsername() {
-        return this.username;
-    }
-
-    public void setUsername(String username) {
-        this.username = username;
-    }
-
-    public String getPassword() {
-        return this.password;
-    }
-
-    public void setPassword(String password) {
-        this.password = password;
-    }
-
-
-    public RegisterModel(String username, String password) {
-        this.username = username;
-        this.password = password;
-    }
-
-}
\ No newline at end of file
+package ohmyquiz.models;
+
+public class RegisterModel {

```

Comment:
cái model này được dùng ở đâu nhỉ? Bình thường chỉ có model của những cái chính (thường là danh từ) Như User, Quiz. Còn đây là action thì k cần model. Mình chỉ dựng data r đưa vào db thôi mà

Sẽ là input (từ UI), ở trên BE sẽ validate (nếu có), và sau khi pass validate thì save vào DB, trả về message thành công. Hoặc failed validate, thì k save vào DB, và trả về message lỗi để UI hiện flash message hoặc alert.

Reply:
ló k đc dùng ở đâu hết ạ tại bạn đầu em hơi lơ mơ cái chia package layout nên tạo á, xong quên xóa